src/vl-core.js

```diff
@@ -155,6 +157,16 @@ export const vlElement = (SuperClass) => {
       }
     }
 
+    /**
+     * Geeft de internationalisatie vertaling terug.
+     *
+     * @param {String} key
+     * @return {String}
+     */
+    getTranslation(key) {
+      return vl.i18n.i18n[key];
+    }
```

Comment:
Misschien kan het nuttig zijn wanneer er geen value wordt gevonden om de key zelf, vb als `???key???` terug te geven. Of een warning te loggen is ook een optie.

Keys zijn strings dus een typo is soms vaak gemaakt en dit kan dan helpen.

Reply:
De caller krijgt undefined terug, dat lijkt me voldoende en duidelijk genoeg dat de vertaling niet bestaat?